release.sh

```diff
@@ -52,6 +52,22 @@ chmod 755 "${BUILDDIR}/update-mgmt-ncn-cfs-config.sh"
 # Copy manifests
 rsync -aq "${ROOTDIR}/manifests/" "${BUILDDIR}/manifests/"
 
+# Copy empty directory created for upgrade using IUF
+rsync -aq "${ROOTDIR}/dummy/" "${BUILDDIR}/dummy/"
+
+# Copy IUF product manifest
+rsync -aq "${ROOTDIR}/iuf-product-manifest.yaml" "${BUILDDIR}/"
```

Comment:
@Pankhuri-Rajesh You can add something like

    yq e ".version = '${RELEASE_VERSION}'" -i "${BUILDDIR}/iuf-product-manifest.yaml"

to get `version` field automatically set to version of CSM release.